runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodePlugin.java

```diff
@@ -208,6 +211,19 @@ public void onOverlayMenuClicked(OverlayMenuClicked overlayMenuClicked)
 		{
 			session.resetRecent();
 		}
+
+		if (overlayMenuClicked.getEntry() == MotherlodeOreOverlay.DISABLED_ENTRY)
```

Comment:
These should be checking the clicked overlay, similar to the reset branch above. In fact, that should be added as a guard clause so none of these code paths can be hit should some other overlay be clicked.